common/src/jni/main/cpp/conscrypt/native_crypto.cpp

```diff
@@ -440,7 +452,7 @@ static void bio_stream_assign(BIO *b, BioStream* stream) {
     b->ptr = static_cast<void*>(stream);
 }
 
-static long bio_stream_ctrl(BIO *b, int cmd, long, void *) {
+static int64_t bio_stream_ctrl(BIO *b, int cmd, int64_t, void *) {
```

Comment:
Revert this. The API you are interacting with uses `long`, so you must use `long`. Your change will break on Windows.

cpplint is correct in complaining about this, but matching the API as specified takes precedence and changing from `long` to `int64_t` sailed long long go.